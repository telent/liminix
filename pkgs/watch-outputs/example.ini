wpa_passphrase={{ output("./example-service","colours/black")  }}
think = {{ string.format("%q", output("./example-service","colours/blue")) }}
argonaut = {{ json_quote "hello\ngoodbye\tnext\027" }}
