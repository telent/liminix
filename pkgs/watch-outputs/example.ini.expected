wpa_passphrase=000000
think = "0000ff"
argonaut = "hello\ngoodbye\tnext\u001B"
